packages/lesswrong/server/notificationTypesServer.tsx

```diff
@@ -39,7 +39,7 @@ export const NewPostNotification = serverRegisterNotificationType({
     if (!post) throw Error(`Can't find post to generate subject-line for: ${postId}`)
     return post.title;
   },
-  emailBody: ({ user, notifications }) => {
+  emailBody: async ({ user, notifications }) => {
```

Comment:
Are these a thing that was particularly integrated with the rest of this PR?

Reply:
Making the revisions collection able to use regular views required integrating it into the permissions system, which required making `checkAccess` functions `async`, and `async` is a plague that spreads through function calls.

Reply:
Gotcha, cool.